packages/typespec-ts/src/transform/transform.ts

```diff
@@ -103,14 +107,76 @@ export async function transformRLCModel(
     apiVersionInfo,
     parameters,
     helperDetails,
-    urlInfo
+    urlInfo,
+    telemetryOptions
   };
 }
 
-export function transformUrlInfo(
-  program: Program,
-  dpgContext: SdkContext
-): UrlInfo | undefined {
+function transformTelemetryOptions(
```

Comment:
Can you put it somewhere else and make the transform.ts file cleaner?

Reply:
Sure.

Reply:
it's still here ? do you forget to move it ?